lib/beaker/cli.rb

```diff
@@ -9,9 +9,7 @@ class CLI
     |   V   |
     |   |   | "
 
-    attr_reader :options, :logger
-
-
+    attr_reader :logger, :options, :network_manager
```

Comment:
I'm also pretty sure that we no longer need to have this as a readable class var.